index.html

```diff
@@ -63,7 +63,110 @@ <h4>Skills</h4>
             <a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a>
             <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
         </div>
-        <hr>                    
+        <span>&#8595; &nbsp;&nbsp; WORK</span>
+        <div class="works-flex">
+            <h1>MY RECENT WORK</h1>
+            <br>
+            <hr>
+            
+            <div class="project-flex">
+                <div class="content-flex">
+                    <div class="layout-flex">
+                        <div>
+                            <img src="./" alt="project-1">
+                        </div>
+                        <h2>Multi-Post Stories</h2>
+                        <p>A daily selection of privately personalized reads; no accounts
+                           or sign-ups required. has been the industry's standard dummy
+                           text ever since the 1500s, when an unknown printer took a 
+                           standard dummy text.
+                        </p>
+                        <ul>
+                            <li>CSS</li>
+                            <li class="border">html</li>
+                            <li class="border">bootstrap</li>
+                            <li class="border">Ruby</li>
+                        </ul>
+                        <button type="submit">
+                            See Project
+                        </button>
```

Comment:
None of your buttons have interaction stylings. Please add them.

<img width="835" alt="Screen Shot 2022-07-07 at 21 44 47" src="https://user-images.githubusercontent.com/82426682/177906640-dd654c7b-9e98-4ebb-83e1-b27e51aa88aa.png">

Reply:
All the buttons have interaction stylings. The last button item carries the property disabled and exhibits the disabled color trait.